### PR TITLE
doc: add leading asterisks for javadoc

### PR DESCRIPTION
**Pull request by Gaurav-Punjabi**
Fixes the violations that are generated after solving issue #6722 
PR Link for Check: [MissingLeadingAsterisk](https://github.com/checkstyle/checkstyle/pull/8085)

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java [hunk 1]
```diff
@@ -184,7 +184,8 @@ public void updateUI() {
                 "Tree.foreground", "Tree.font");
     }
 
-    /* Workaround for BasicTableUI anomaly. Make sure the UI never tries to
+    /**
+     * Workaround for BasicTableUI anomaly. Make sure the UI never tries to
```
Comment (strkkk) [inline src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java:188]: I dont think this is a valid fix. MissingAsterisk checks only javadocs, but it is not a javadoc comment.
Same for other cases below

Comment (Gaurav-Punjabi) [reply src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java]: Done, updated.

**Comment by Gaurav-Punjabi** [comment]
> Please rename commit message, now it is incorrect.

Done :)

**Comment by strkkk** [comment]
ok to merge after ci pass

**Comment by Gaurav-Punjabi** [comment]
@strkkk New violations were generated, after updating the check according to the following comment [checkstyle/checkstyle#8085](https://github.com/checkstyle/checkstyle/pull/8085#discussion_r410207230). So I fixed them and updated the PR. Please review.

**Pull request merged**